### PR TITLE
CORTX-32704: Minor Changes and added Tested By for Community Build deployment

### PR DESCRIPTION
**Pull request by mukul-seagate11**
Signed-off-by: Mukul Malhotra <mukul.malhotra@seagate.com>

# Problem Statement
- Minor Changes and added Tested By

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM
- [ ] Jenkins pipelines used for testing

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [x] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [x] Changes done to WIKI / Confluence page / Quick Start Guide

-----
[View rendered solutions/community-deploy/CORTX-Deployment.md](https://github.com/Seagate/cortx-re/blob/mukul-seagate11-patch-1/solutions/community-deploy/CORTX-Deployment.md)

### CODE REVIEW

**Comment by mukul-seagate11** [comment]
PR owner added in cla list

**Review by shailesh-vaidya — DISMISSED** [review]

File: solutions/community-deploy/CORTX-Deployment.md [hunk 1]
```diff
@@ -70,19 +70,19 @@ hostname=cortx-deploy-node2.cortx.com,user=root,pass=<root-password>
 hostname=cortx-deploy-node3.cortx.com,user=root,pass=<root-password>
 ```
 
--  Execute `cluster-setup.sh` to setup K8s cluster on your VM for N-node deployment.
--  To allow PODs creation on primary node pass the first input parameter for `cluster-setup.sh` script as true. Please note you must pass the input parameter as true for Single Node Setup.
+-  Execute `cluster-setup.sh` script from primary node to setup K8s cluster on your EC2 instances for N-node deployment.
+-  To allow PODs creation on primary node to pass the first input parameter for `cluster-setup.sh` script as true. Please note you must pass the input parameter as true for N-node setup.
```
Comment (gauravchaudhari02) [inline solutions/community-deploy/CORTX-Deployment.md:74]: I guess `,` is required in place of `to`

Comment (shailesh-vaidya) [inline solutions/community-deploy/CORTX-Deployment.md:74]: Please add any N Node related details in https://github.com/Seagate/cortx-re/tree/f-94l-multinode

File: solutions/community-deploy/CORTX-Deployment.md [hunk 2]
```diff
@@ -70,19 +70,19 @@ hostname=cortx-deploy-node2.cortx.com,user=root,pass=<root-password>
...
 
 ```
-# ./cluster-setup.sh true
+./cluster-setup.sh true
 ```
 
 ## Deploy CORTX Stack 
 
-Execute `cortx-deploy.sh` to deploy the CORTX on your K8s Cluster.
+Execute `cortx-deploy.sh` script from primary node to deploy the CORTX stack on your K8s cluster.
```
Comment (shailesh-vaidya) [inline solutions/community-deploy/CORTX-Deployment.md:82]: We will need instructions to identify a primary node. Please add any N Node related details in https://github.com/Seagate/cortx-re/tree/f-94l-multinode

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]

File: solutions/community-deploy/CORTX-Deployment.md (hunk 1 above)
Comment (mukul-seagate11) [reply solutions/community-deploy/CORTX-Deployment.md:74]: Done

File: solutions/community-deploy/CORTX-Deployment.md (hunk 2 above)
Comment (mukul-seagate11) [reply solutions/community-deploy/CORTX-Deployment.md:82]: Its addressed in https://github.com/Seagate/cortx-re/pull/1156

File: solutions/community-deploy/CORTX-Deployment.md (hunk 1 above)
Comment (mukul-seagate11) [reply solutions/community-deploy/CORTX-Deployment.md]: fixed

File: solutions/community-deploy/CORTX-Deployment.md [hunk 3]
```diff
@@ -14,80 +14,62 @@ Following are the minimum system specifications required to set up the K8s clust
...
-
 ```
-# reboot
+reboot
 ```   
 
-### Note
- 1. All the nodes should be reachable over SSH with root user.
+**Note:**
+ 1. All the EC2 nodes should be reachable over SSH
```
Comment (shailesh-vaidya) [inline solutions/community-deploy/CORTX-Deployment.md:44]: Please remove EC2. These instructions are generic for VM's created on any platform

File: solutions/community-deploy/CORTX-Deployment.md [hunk 4]
```diff
@@ -14,80 +14,62 @@ Following are the minimum system specifications required to set up the K8s clust
...
 ```
-# ./cluster-setup.sh true
+./cluster-setup.sh true
 ```
 
 ## Deploy CORTX Stack 
-
-Execute `cortx-deploy.sh` to deploy the CORTX on your K8s Cluster.
-
+- Execute `cortx-deploy.sh` script from primary node to deploy the CORTX stack on your K8s cluster.
```
Comment (shailesh-vaidya) [inline solutions/community-deploy/CORTX-Deployment.md:66]: Please remove the `primary node`. This should be part of N node PR

File: solutions/community-deploy/CORTX-Deployment.md [hunk 5]
```diff
@@ -105,19 +87,18 @@ Following parameter/s are passed when the cluster deployment command executes. I
 | NAMESPACE  | default | Kubernetes cluster Namespace for CORTX deployments. |
 | SOLUTION_CONFIG_TYPE | manual | There are two config types for solution.yaml file; manual and automated. In automated type the solution.yaml is created by script if VM is created as per standard specification. In manual type the user needs to create solution.yaml with required disks, image details etc.; place it at script location and configure SOLUTION_CONFIG_TYPE variable as manual. |
 
-
-For example:
+**For example:**
 ```
-# export CORTX_SCRIPTS_BRANCH=integration && export CORTX_SCRIPTS_REPO=AbhijitPatil1992/cortx-k8s && export SOLUTION_CONFIG_TYPE=automated && ./cortx-deploy.sh --cortx-cluster
+export CORTX_SCRIPTS_BRANCH=integration && export CORTX_SCRIPTS_REPO=<owner>/cortx-k8s && export SOLUTION_CONFIG_TYPE=automated && ./cortx-deploy.sh --cortx-cluster
```
Comment (shailesh-vaidya) [inline solutions/community-deploy/CORTX-Deployment.md:92]: Please replace `<owner>` with `Seagate`

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]
Please remove `Multi-node` from PR title

**Comment by mukul-seagate11** [comment]
> Please remove `Multi-node` from PR title

Done

File: solutions/community-deploy/CORTX-Deployment.md (hunk 3 above)
Comment (mukul-seagate11) [reply solutions/community-deploy/CORTX-Deployment.md]: done

File: solutions/community-deploy/CORTX-Deployment.md (hunk 5 above)
Comment (mukul-seagate11) [reply solutions/community-deploy/CORTX-Deployment.md]: done

**Review by shailesh-vaidya — APPROVED** [review]

**Review by gauravchaudhari02 — APPROVED** [review]

**Pull request merged**